i18n/zh/docusaurus-plugin-content-docs/current/administrator/upgrading/v1.5-v1.6.md

```diff
@@ -0,0 +1,33 @@
+---
+title: v1.5 升级到 v1.6
+---
+
+遵循[常规升级流程](./README.md)。
+
+## 升级后显著变化
+
+### API变化
+
+* PropagationPolicy 中  `AffinityName` 的长度现在限制为 (1, 32]，并且必须是合规名称。
```

Comment:
![c1253dbe2e7afaa81a94022f2fcd11c](https://github.com/karmada-io/website/assets/7522079/537ed1e7-8359-4014-a199-cb48b8b9152e)
It seems [1, 32], right?

Reply:
Thanks, update.